app/Helpers/TopicSupport.php

```diff
@@ -598,6 +642,12 @@ public static function reorderSupport($orders, $topicNum, $allNickNames)
                     ->whereIn('nick_name_id', $allNickNames)
                     ->update(['support_order' => $order['order']  // update your field(s) here
                         ]);
+                    
+                    /* To update the Mongo Tree while adding support - CAN-1042 */
+                    /* We need to update the execute the job for reordering as well, only when topicnumber == 81 */
```

Comment:
I didn't understand this comment? What does it mean?

Reply:
This comment means that we are updating mongo tree when user will add new support. 
And the second line means that there are multiple camp numbers here, so we are using the dynamic camp number for topic 81 only at canonizer service. thats why i mentioned this in comment.